### DISCUSSION TITLE
Backslash is not working

### DISCUSSION THREAD

**Discussion by Sakura0134** [AppGround-io/bluetooth-keyboard-and-mouse-support#189] [Bugs & Problems]
The backslash(\) and its complement dash(The straight line, | ) is not giving proper input in Gnome, Fedora(Linux). Instead, it is printing > and < . Android device Vivo y15, OS: Android 11, Keyboard: Hackers Keyboard(en-us qwerty), but is reproducible in any software keyboard like gboard. Also, it works fine in the UK keyboard layout and other keyboard layouts.(like English Canada)

**Comment by normanr** [comment]
I found `English US Intl` to be a reasonable workaround. It sends `\` and `|` correctly and also has the correct mapping for most of the other symbols.
